crates/bevy_picking/src/lib.rs

```diff
@@ -239,7 +238,6 @@ impl Plugin for PickingPlugin {
                     // Eventually events will need to be dispatched here
                     PickSet::Last,
                 )
-                    .ambiguous_with(bevy_asset::handle_internal_asset_events)
```

Comment:
I'm curious why this was possible to remove :)

Reply:
We ignored all ambiguities with `handle_internal_asset_events` 😆